framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -7445,8 +7450,33 @@ VkResult VulkanReplayConsumerBase::OverrideCreateSwapchainKHR(
 
         if (!surface_format_supported)
         {
+            // handle VK_KHR_swapchain_mutable_format
+            if (modified_create_info.flags & VK_SWAPCHAIN_CREATE_MUTABLE_FORMAT_BIT_KHR)
+            {
+                bool mutable_srgb = false;
+                if (auto* format_list_pnext =
+                        graphics::vulkan_struct_get_pnext<VkImageFormatListCreateInfo>(&modified_create_info))
+                {
+                    for (uint32_t i = 0; i < format_list_pnext->viewFormatCount; ++i)
+                    {
+                        if (vkuFormatIsSRGB(format_list_pnext->pViewFormats[0]))
+                        {
+                            mutable_srgb = true;
+                            break;
+                        }
+                    }
+
+                    // overwrite existing VkImageFormatListCreateInfo
```

Comment:
Nit, this should be a warning (maybe just once) that you are overwritting the swapchain format.